src/utils/moduleId.js

```diff
@@ -3,5 +3,8 @@ import global from './global'
 const ns = '2f1acc6c3a606b082e5eef5e54414ffb'
 if (global[ns] == null) global[ns] = 0
 
-// In case we have more than one module version.
+// Bundle may contain multiple JSS versions at the same time. In order to identify
```

Comment:
I still don't get it. How is the bundle containing multiple JSS versions on issue? Doesn't the issue coming from the different JSS instance?

Reply:
If bundle contains multiple JSS package versions, modules are evaluated separately, which means JSS instance counter will start from 0 for each version.

Reply:
I could replace the module id with a JSS instance counter which always starts using a global counter, like I do with moduleId right now, so there will be no need for module id. I thought this implementation will be easier to reason about.